### ISSUE TITLE
multiscaleEPE 

### ISSUE THREAD

**Issue by shanchao0906** [ClementPinard/FlowNetPytorch#104]
Code training used in the process:
loss = multiscaleEPE(output, target, weights=args.multiscale_weights, sparse=args.sparse)
flow2_EPE = args.div_flow * realEPE(output[0], target, sparse=args.sparse)
I can't understand the loss function you designed, could you please explain it?

**Comment by Jamesgender** [comment]
> 

Actually the EPE is not designed by author, it is a common index just like RMSE. In original Paper, the author writes"Training loss we use the endpoint error the Euclidean distance between the predicted flow vector and the ground truth, averaged over all pixels" That is.
